packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -77,6 +77,9 @@ export class FileSystemFrontendContribution implements FrontendApplicationContri
     @inject(FileService)
     protected readonly fileService: FileService;
 
+    protected editorFileWasChangedEmitter = new Emitter<{ editor: NavigatableWidget, type: FileChangeType }>();
+    readonly onEditorFileChanged = this.editorFileWasChangedEmitter.event;
```

Comment:
I'm fine with https://github.com/eclipse-theia/theia/pull/10361/files#r747644822 and keeping the events in the contribution, only comment would be if we have to update the naming of the events following the [coding guideline](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#event_names) but I'll leave it to your discretion :+1:

Reply:
Thanks for the feedback! I've pushed a change that addresses your comments